src/app/services/hero/hero.service.ts

```diff
@@ -0,0 +1,20 @@
+import { Injectable } from '@angular/core';
+
+import { HeroModel } from 'models/animal/hero/hero.model';
```

Comment:
Fix import

`import { HeroModel } from 'models/animal/hero/hero.model';`

to

`import { HeroModel } from 'models/hero/hero.model';`

Reply:
hahahaha man!! i didn't notice haha